lib/Migrate_ast.mli

```diff
@@ -1,3 +1,14 @@
+(**********************************************************************
+ *                                                                    *
+ *                            OCamlFormat                             *
+ *                                                                    *
+ *  Copyright (c) 2017-present, Facebook, Inc.  All rights reserved.  *
```

Comment:
should be the current year I guess

Reply:
Ah, actually I haven't gotten around to it, but we don't need years at all now. We could either remove them all in one commit or just not add them or...

Reply:
If you can update them that'd be great. thanks!